proto/feeburner/params.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package neutron.feeburner;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/neutron-org/neutron/x/feeburner/types";
+
+// Params defines the parameters for the module.
+message Params {
+  option (gogoproto.goproto_stringer) = false;
+  // Defines Neutron denom
+  string neutron_denom = 1;
+  // Defines Gaia denom
+  string gaia_denom = 2;
+  // Defines Treasury address
+  string treasury_address = 3;
```

Comment:
do we need treasury address here? What for? for informational purpose only?

Reply:
Well, if you say that treasury address should be in genesis, I will remove it from params then.

Reply:
Sorry for confusing. No need to remove it from params, because it should be set via params in the genesis, and can be updated later via PramsChange proposal. My question was about do we really need this information at all? I saw that you are sending coins from "Gaia" demon to the contract, but is this required? Maybe we can choose other mechanics?

Reply:
This implementation complies with the specification: https://www.notion.so/Design-Minimal-Design-v0-6-1-59dac6d17c5e4505bf2c9786425d86bb#f9af00c053bb4ab9b704bab100844cb8

(the treasury is a smart-contract)